src/NuGet.Clients/PackageManagement.VisualStudio/SolutionRestore/SolutionRestoreWorker.cs

```diff
@@ -53,7 +59,28 @@ internal sealed class SolutionRestoreWorker : ISolutionRestoreWorker, IDisposabl
             _serviceProvider = serviceProvider;
 
             _componentModel = _serviceProvider.GetService<SComponentModel, IComponentModel>();
+#if VS15
+            _solutionManager = _componentModel.GetService<IVsSolutionManager>();
```

Comment:
But this is not an mef import or dependency... SolutionManager is NuGet internal interface and its implementation. I still don't understand why its required...